firefox_puppeteer/errors.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.errors import *
```

Comment:
I would leave them separate. If we need our own exceptions, that's fine, but marionette's always correspond to something directly from the protocol its using to talk to the browser, accessing them from the same place might just cause confusion.